app/policies/user_policy.rb

```diff
@@ -76,7 +78,9 @@ def resolve
       if current_user.superadmin?
         scope.web_users
       elsif current_user.admin?
-        scope.web_users.where(role: %w(admin organisation_admin normal))
+        scope.web_users.where(role: %w(admin super_organisation_admin organisation_admin normal))
```

Comment:
Could we use `where(role: current_user.role.manageable_roles)` here (and in other branches of the condition).  I don't think we're making much use of `manageable_roles` and it makes sense to try and keep that stuff as centralised as possible if we can.